go.mod

```diff
@@ -14,8 +14,9 @@ require (
 	github.com/rs/cors v1.7.0
 	github.com/shirou/gopsutil v3.20.12+incompatible
 	github.com/sirupsen/logrus v1.7.0 // indirect
-	github.com/stretchr/testify v1.7.0 // indirect
+	github.com/stretchr/testify v1.5.1
```

Comment:
Any specific reason for a downgrade here?

Reply:
I just did go mod tidy and it automatically happened. I will upgrade the version and push the changes.

Reply:
@aquibbaig I have upgraded the version and pushed the changes.

Reply:
I think we should follow what the mod says. It's generally more stable, but I will not block this PR for this change.

Reply:
+1